dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -78,6 +78,9 @@ sslRef=SSL reference
 sslRef$Ref=SSL reference
 sslRef.desc=ID of the SSL configuration to be used to connect to the SSL-enabled LDAP server.
 
+loginProperty=Login property
+loginProperty.desc=The name of the login property
```

Comment:
Can you add a period after "he name of the login property" to match the style of the other descriptions in the file?